### DISCUSSION TITLE
New Wireless not working.

### DISCUSSION THREAD

**Discussion by Map3GOSM** [MiczFlor/RPi-Jukebox-RFID#2180] [Q&A]
My brother-in-law and I built a PhonieBox for my son. We set everything up at his house using his wireless network, and everything worked fine. We added my wireless network through the web interface. However, when I took it home, the PhonieBox wouldn't connect to my wireless network. If I change my phone's hotspot to match the SSID and password of my brother-in-law's network, the box connects to my phone. We tried the same at my parents' house, and encountered the same issue. Any tips would be appreciated. Thanks, and happy holidays!

**Comment by s-martin** [comment]
Please open a ssh connection to the Phoniebox and check and set the WiFi settings in the raspberry configuration with `sudo raspi-config`.

See here for details https://www.raspberrypi.com/documentation/computers/configuration.html

**Reply by Map3GOSM** [reply]
I think since I am using a USB-Wireless dongle the raspi-config does not show me any wireless options. I set wireless up following this instruction: https://youtu.be/oM2kAnITNyE?si=6O24cBKoCmROWziN . It works fine with the initial set Input but if I change it it doesn't connect at all and when I set it back it to the initial wireless I can connect to the Raspberry again.

**Reply by AlvinSchiller** [reply]
If I understood right you used the webui first and later the instruction from the video?

Can you supply the content of  your `/etc/wpa_supplicant/wpa_supplicant.conf` and `/etc/dhcpcd.conf` with masked password?

**Reply by Map3GOSM** [reply]
Sure. Here is the dhcp.conf:

[dhcpcd.conf.txt](https://github.com/MiczFlor/RPi-Jukebox-RFID/files/13773383/dhcpcd.conf.txt)


And here the wpa_suplicant.conf:

[wpa_supplicant.conf.txt](https://github.com/MiczFlor/RPi-Jukebox-RFID/files/13773384/wpa_supplicant.conf.txt)

Following the YouTube Tutorial I set up the following two files:
etc/network/interfaces

[interfaces.txt](https://github.com/MiczFlor/RPi-Jukebox-RFID/files/13773386/interfaces.txt)

and etc/wpa.conf

[wpa.conf.txt](https://github.com/MiczFlor/RPi-Jukebox-RFID/files/13773391/wpa.conf.txt)

Looking at it now, I realize why adding it through the web interface didn't work. I assume this is because the interfaces file refers to the wpa.conf file, not the wpa_supplicant.conf file. I attempted to modify the interfaces file to link to the wpa_supplicant.conf file, but that was unsuccessful. I also tried changing the wpa.conf file to my own wireless network, which also didn't work. The only time it accepts the settings is when I revert it back to 'WiFip,' which is the wireless network of my brother-in-law. I added the .txt endings otherwise GitHub wouldn't allow the upload.

**Reply by AlvinSchiller** [reply]
I have some more questions.

- Which RaspberryPi OS Version do you use?
- Did you have a wired connection before you added the wifi via webui or was it already set in the Rpi Imager?

**Reply by Map3GOSM** [reply]
I have the lite version installed and initially installed the Wifi via the Imager. Since we originally tested on a zero w everything worked fine. We then switched the SD card to the zero without Wireless and followed the video instructions to configure the Dongle additionally.

**Reply by AlvinSchiller** [reply]
I meant version like bullseye or bookworm? 
🙂

**Reply by Map3GOSM** [reply]
Bullseye sorry 😊

**Reply by AlvinSchiller** [reply]
Can you try and remove the wpa.conf and all entries in the interfaces.conf except the first 3 lines?
Then go to the phonieboxUi and set your Wifi again.
Then reboot

**Reply by Map3GOSM** [reply]
If I remove the wpa.conf files the raspberry won't connect to the Wifi and then I cant connect to the WebUI ... or am I missing something?

**Reply by AlvinSchiller** [reply]
If you make it in one session, its ok, as it will only reload after the reboot. 
But if its still not working after the reboot you would need a LAN connection or Monitor, true.

**Reply by Map3GOSM** [reply]
Ahh ok. I will try that. But I will be on the road back home today so I probably won't get to it till tomorrow.

**Reply by Map3GOSM** [reply]
Hi Sorry for the late response. Did not get around to it over New Year. I played around a little bit more  with no success sadly. I ultimately capitulated and ordered a new Pi with integrated Wireless. Without doing anything else I inserted the SD card into the new Pi and everything is working flawlessly. I assume it was an Issue with the wireless Dongle. The old Pi will find a new Home somewhere where I don't need Wireless. Thank You still Very much for the Support.

**Comment by pabera** [comment]
You can set the wpa_supplicant.conf also without the RPi.

See [this](https://www.thedigitalpictureframe.com/three-ways-to-change-your-wifi-password-and-ssid-on-a-headless-raspberry-pi-locally-and-remotely/).

**Reply by Map3GOSM** [reply]
I tried this as well with no success. As I stated I believe this was an Wireless Dongle Issue.
